Added Vision statement (#56)
wildfirepy/wildfirepy: Vision_Statement.md

# WildfirePy's Vision Statement

## About WildfirePy

> WildfirePy is a community-developed and community-driven free and open-source Python package that provides functionality for wildfire location, tracking and simulation in a single code-base.
> 

Reviewer: ```suggestion
WildfirePy is a community-developed and community-driven, free and open-source Python package for wildfire data analysis, tracking and simulation.

```

Wildfires are devasting phenomena which cause loss of humongous volumes and variety of flora and fauna owing to their uncontrolled nature. They also pose a threat to the quality of life for humans, constituting the hazards of suffocation, burns, injuries and air pollution leading to a range of health issues. Wildfires have a significant and detrimental impact on the weather and climate by releasing large quantities of carbon dioxide, carbon monoxide and fine particulate matter into the atmosphere.

Scientific organizations worldwide have been working towards forecasting and preventing this calamity. They have collected colossal databases recording wildfires, their triggers and repercussions -- most of which are publicly available. However, this information is often localized and stored in inconsistent formats. This is where WildfirePy comes in.

WildfirePy can be used to collect data from these sources for analysis. This package can be used to visualize this data as well as track and simulate wildfires. WildfirePy seeks to play its part in the prevention, detection, control and suppression of wildfires.

## Community Code of Conduct

> TBD

Reviewer: We have a Code of Conduct.

> WildfirePy aims to collect data from these sources and pack it into a consistent and cohesive code-base. This package can be used to visualize this data as well as track and simulate wildfires. WildfirePy seeks to play its part in the prevention, detection, control and suppression of wildfires.
> 

Reviewer: ```suggestion
WildfirePy can be used to collect data from these sources for analysis. This package can be used to visualize this data as well as track and simulate wildfires. WildfirePy seeks to play its part in the prevention, detection, control and suppression of wildfires.

```

## Organizational Structure

Will be added after WEP 001 is up.

## Development Procedure

> TBD

Reviewer: @abhijeetmanhas  @kakirastern What should we add here?

## Programming Guidelines

> 

Reviewer: Add a link to CONTRIBUTING here, with a sentence explaining the file

### Choice of Languages

> 

Reviewer: Add python

### Versioning

### Dependencies

> 

Reviewer: You can add this based on the deps in the setup.cfg file.

## Community Code of Conduct

> Files detailing the PR and issue templates can be found [here.](https://github.com/wildfirepy/wildfirepy/tree/master/.github)

Reviewer: ```suggestion
We encourage contributors to look up the [files detailing the PR and issue templates](https://github.com/wildfirepy/wildfirepy/tree/master/.github)
```

TBD

## Programming Guidelines
The [CONTRIBUTING.md](https://github.com/wildfirepy/wildfirepy/blob/master/CONTRIBUTING.md) file contains guidelines for contributing to the organization as well as opening issues and making pull requests.

### Choice of Languages
Python

### Versioning

### Dependencies
Python >=3.5\
setuptools_scm\
pyproj\
requests\
pytest\
pytest-cov\

> pytest-remotedata\

Reviewer: There should be a separate subsection for testing deps and other dependencies.
